qubesadmin/tools/qvm_create.py

```diff
@@ -113,6 +121,14 @@ def main(args=None, app=None):
     if args.one_pool:
         pool = args.one_pool
 
+    if 'disp' in args.properties:
```

Comment:
```suggestion
    if args.disp:
```

Reply:
I was going for a consistent coding-style with this. If this should be changed, it should be changed across the whole file, which would constitute another pull request IMO?